R/utils.R

```diff
@@ -2,6 +2,17 @@ last <- function(x) x[length(x)]
 
 sq <- function(x) glue::single_quote(x)
 
+sglue <- function(...) stop(glue::glue(..., .envir = parent.frame()), call. = FALSE)
```

Comment:
Do as you see in the glue issue. BTW, at that time it was written `f()` is what we now call `glue()`.